hooks/conan-center.py

```diff
@@ -76,22 +82,26 @@ def _get_message(self, message):
             return message
 
     def success(self, message):
-        self._output.success(self._get_message(message))
+        if self._logging_level < WARNING:
+            self._output.success(self._get_message(message))
 
     def debug(self, message):
         if self._error_level and self._error_level <= DEBUG:
             self._error = True
-        self._output.debug(self._get_message(message))
+        if self._logging_level < INFO:
```

Comment:
probably it should be `if self._logging_level <= WARNING:` ?

I actually don't like the code need to check level before the output.
I guess these functions (info/warning/error) should check the level itself, and render no-op if level is too high

Reply:
> probably it should be if self._logging_level <= WARNING: ?

Not for debug messages. It's lower level than INFO:

https://docs.python.org/3/library/logging.html#levels

> I actually don't like the code need to check level before the output.

Me too, but it's a logger from Conan client, we would need initialize a new one only for hooks, but also I dunno if it's a singleton.

Reply:
I'm thinking this should be:
```suggestion
        if self._logging_level <= DEBUG:
```